sisl/io/siesta/ani.py

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""
+Sile object for reading ANI files
+"""
+from ..xyz import xyzSile
+from ..sile import add_sile
+from sisl._internal import set_module
+
+__all__ = ["aniSileSiesta"]
+
+@set_module("sisl.io.siesta")
+class aniSileSiesta(xyzSile):
+
+    def read_geometry(self, *args, **kwargs):
```

Comment:
I would just change default argument in the argument list here, no need to check in the code.

Reply:
I made another suggestion. Is this what you had in mind?

Reply:
It could be simpler, simply do:
```python
read_geometry(self, *args, *, all=True, **kwargs)
```

Reply:
Sorry, I tried different variations of this but it failed (I'm afraid I miss something with the syntax).

Reply:
Yeah, my suggestion was wrong, I'll make a PR against your branch, then you can see.